src/components/DetailViewHeader.jsx

```diff
@@ -95,6 +104,15 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
+            <Button className={classes.exportButton} variant="contained" color="primary">
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href={`${terraUrl}/#import-data?url=${URL_ENCODED_TDR_ROOT_URL}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
```

Comment:
is there a reason we're breaking out the TDR root url and not just using `window.location.origin` here?